src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -82,14 +109,41 @@ const OfflineKeyShareGeneration = () => {
         setSelectedBox(id);
     };
 
+    const changeWithdrawalAddressHandler = (e: any) => {
+        const { value } = e.target;
+        setWithdrawalAddress(value);
+        validateAddressInput(value, setAddressValidationError);
+        setTextCopied(false);
+        setConfirmedWithdrawalAddress(false);
+    };
+
+    const showCopyButtonCondition = selectedBox === 1 || (selectedBox === 3 && withdrawalAddress && !addressValidationError.shouldDisplay && confirmedWithdrawalAddress);
```

Comment:
Sorry I didn't see these numbers here, can you do here the same as in first PR (use enum or constants)

Reply:
done